src/talk/renderer/TalkDesktop.app.ts

```diff
@@ -15,8 +15,11 @@ export async function createTalkDesktopApp() {
 
 	// Load Talk Desktop asynchronously to make sure,
 	// no module is loaded before the page has been set up and ready to run apps
-	const { default: TalkDesktop } = await import('./TalkDesktop.vue')
+	const TalkDesktopApp = defineAsyncComponent(() => import('./TalkDesktop.vue'))
```

Comment:
There was async import already, works fine without an additional component wrapper.

Reply:
I wanted to unify it between all windows, but if you say it's not needed, let's drop it

Reply:
If unify, I'd drop it everywhere then: to have fewer wrappers in the component tree